service/storage_service.cc

```diff
@@ -1702,16 +1689,15 @@ storage_service::prepare_replacement_info(std::unordered_set<gms::inet_address>
     }
 
     auto dc_rack = get_dc_rack_for(replace_address);
-    auto raft_id = get_raft_id_for(_gossiper, replace_address);
 
     auto replace_host_id = _gossiper.get_host_id(replace_address);
-    slogger.info("Host {}/{} is replacing {}/{} using a different address", _db.local().get_config().host_id, get_broadcast_address(), replace_host_id, replace_address);
+    slogger.info("Host {}/{} is replacing {}/{}", _db.local().get_config().host_id, get_broadcast_address(), replace_host_id, replace_address);
```

Comment:
Sneaked-in a fix for the message (https://github.com/scylladb/scylladb/pull/12250#discussion_r1045626954)

Reply:
ack